crypto/http/http_client.c

```diff
@@ -355,6 +357,16 @@ int OSSL_HTTP_REQ_CTX_set1_req(OSSL_HTTP_REQ_CTX *rctx, const char *content_type
     return res;
 }
 
+void OSSL_HTTP_REQ_CTX_set_max_response_hdr_lines(OSSL_HTTP_REQ_CTX *rctx,
+                                               unsigned long count)
```

Comment:
`max_hdr_lines` is an `unsigned int`, but this is an `unsigned long`. Probably shouldn't use `long`